core/point/point.go

```diff
@@ -1,13 +1,40 @@
 // Package point is a basic package for points.
 package point
 
+import (
+	"fmt"
+)
+
 // Point is a basic point. Although simple, the member variables are kept
 // private to ensure that Point remains immutable.
 type Point struct {
 	x int64
 	y int64
 }
 
+// pointToSgfMap is a translation reference between int64 Point
+// and string SGF-Point (rune) values
+var pointToSgfMap = map[int64]rune{
+	0: 'a', 1: 'b', 2: 'c', 3: 'd', 4: 'e', 5: 'f', 6: 'g',
+	7: 'h', 8: 'i', 9: 'j', 10: 'k', 11: 'l', 12: 'm', 13: 'n',
+	14: 'o', 15: 'p', 16: 'q', 17: 'r', 18: 's', 19: 't', 20: 'u',
+	21: 'v', 22: 'w', 23: 'x', 24: 'y', 25: 'z', 26: 'A', 27: 'B',
+	28: 'C', 29: 'D', 30: 'E', 31: 'F', 32: 'G', 33: 'H', 34: 'I',
+	35: 'J', 36: 'K', 37: 'L', 38: 'M', 39: 'N', 40: 'O', 41: 'P',
+	42: 'Q', 43: 'R', 44: 'S', 45: 'T', 46: 'U', 47: 'V', 48: 'W',
+	49: 'X', 50: 'Y', 51: 'Z',
+}
+
+// sgfToPointRef is an anonymous function that runs and inverts the
+// pointToSgfMap to create the reverse translation reference
+var sgfToPointRef = func(mapIn map[int64]rune) map[rune]int64 {
```

Comment:
in general, in go, you shouldn't name things blahMap or blahSlice. That embeds the type in the variable name - but that's unnecessary.

so instead of blahMap just use blah, instead of mapIn, mapElem, you even can just use `in`, `out`.